data/family/Debian.yaml

```diff
@@ -0,0 +1,10 @@
+---
+kafka::user_shell: '/usr/sbin/nologin' 
```

Comment:
I don't think that all those data duplication is perfect, but I think it can be cleaned up in the future when all/more is moved to hiera and we've a better ida of the general structure.

Reply:
I agree. This is related to stuff like: 
* https://github.com/voxpupuli/puppet-kafka/blob/master/manifests/broker/install.pp#L10-L30